csv/readcsv.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	runtime "github.com/ahsanfayaz52/EmployeeManagementSystem"
+	"github.com/ahsanfayaz52/EmployeeManagementSystem/models"
+)
+
+func main() {
+	rt, err := runtime.NewRuntime()
+	if err != nil {
+		panic(err)
+	}
+	root, err := os.Getwd()
+	if err != nil {
+		log.Fatal(err)
+	}
+	filepath := "/Files/records.csv"
+	path := root + filepath
+	var rows = ReadCsv(path)
+	AddDataToDatabase(rows, rt)
+}
+
+// AddDataToDatabase adds employee data from csv to database.
+func AddDataToDatabase(rows [][]string, rt *runtime.Runtime) string {
```

Comment:
Just name the function `import` and why are exporting the function? It's not going to be used outside the package.

Reply:
import is reserved in golang so i used importData instead